FAQ.md

```diff
@@ -0,0 +1,113 @@
+
+Once your Lightning wallet has been funded and your channel opened, the application will give you the ability to trade financial derivatives directly from your wallet, and even hold synthetic cash, such as USDp, the first USD on Lightning.
+
+### On-chain? Lightning?
+
+On-chain is a term used to designate the traditional Bitcoin network : It is used for its high level of security and verifiability, and allows large amounts of money to be sent and received for a fee to register transactions in the blockchain.
+
+Lightning is a payment network built on top of Bitcoin (referred as layer), designed to enable instant and practically free transactions once your node and channel are set up, while benefiting from the security of Bitcoin's legacy network.
+
+### Will 10101 manage the funds for me?
+
+No, at no time will 10101 hold your bitcoins.
+
+Our application makes a point of respecting and educating users in the principles of self-custody : you hold your own keys, you remain responsible for your wallet and funds, and you retain full control throughout your experience.
+
+That's why you need to make sure you back up your recovery phrase and keep it safe.
+
+### How is it possible to trade directly from my wallet, without a platform?
+
+This prowess is made possible by Discreet Log Contracts.
+
+These are peer-to-peer contacts, involving two parties who lock funds into a specific channel. The release of which is conditional: when the contract expires, it releases the funds in the proportions corresponding to the conditions defined at the outset, using data from an oracle to ensure that the transaction proceeds smoothly, in order to prevent any non-collaborative outcome.
+
+
+DLCs can be seen as an additional layer built on top of the Lightning Network.
+
+If you want to learn more, we produced a series of [blog posts](https://10101.finance/blog/dlc-to-lightning-part-1) on this topic, and another one on the specific case of [trading settlements](https://10101.finance/blog/trade-settlement).
+
+
+### How do USDp and synthetic currencies work?
+
+USDp and synthetic currencies within 10101 are referred to synthetics in finance.
+You will have a USD balance circulating on Lightning, holding nothing but sats and a short position combined.
+
+It's a financial product that hedges you against the volatility of Bitcoin and tracking the value of USD, using only Bitcoin.
+
+If you want to learn more about it we've produced a [blog post](https://10101.finance/blog/synthetic-stable) on this topic.
+
+### What's the point of the recovery phrase?
+
+Your recovery phrase (sometimes called "seed") is the only way to redeem your funds. If you've deleted the app or lost the data on your phone, you will not be able to access your funds without it. We will be unable to assist you either.
```

Comment:
I'm not very inspired right now, but I think saying that the seed is the "only way to redeem your funds" sounds a bit weird. Maybe we can rearrange this to make it clear that the seed is only needed _if_ something goes wrong.